core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/relational/ComparisonFunctionExpression.kt

```diff
@@ -0,0 +1,36 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.relational
+
+import ch.ergon.dope.resolvable.expression.UnaliasedExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.FunctionExpression
+import ch.ergon.dope.validtype.ComparableType
+
+sealed class ComparisonFunctionExpression<T : ComparableType>(
+    symbol: String,
+    firstExpression: UnaliasedExpression<T>,
+    secondExpression: UnaliasedExpression<T>,
+    vararg additionalExpressions: UnaliasedExpression<T>,
+) : FunctionExpression<T>(symbol, firstExpression, secondExpression, *additionalExpressions)
+
+class GreatestExpression<T : ComparableType>(
```

Comment:
I would move Greatest and Least each in a seperate file

Reply:
done